addons/website/views/snippets/s_masonry_block.xml

```diff
@@ -3,63 +3,58 @@
         </div>
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-5 text-center o_cc o_cc2" data-name="Block"
-            style="grid-area: 1 / 7 / 6 / 10; z-index: 2;" >
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+        <div class="o_grid_item o_cc o_cc3 g-height-4 g-col-lg-4 col-lg-4 justify-content-end rounded-4" data-name="Block" style="--grid-item-padding-y: 20px; --grid-item-padding-x: 20px; z-index: 3; grid-area: 1 / 9 / 5 / 13; background-image: linear-gradient(135deg, var(--o-color-4) -400%, var(--o-color-2) 100%);">
+            <h3>Greater <br class="d-none d-lg-inline"/>Impact</h3>
+            <p>Making a difference every day</p>
         </div>
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-5 text-center o_cc o_cc3" data-name="Block"
-            style="grid-area: 1 / 10 / 6 / 13; z-index: 3;">
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+        <div class="o_grid_item o_cc o_cc2 g-height-4 g-col-lg-3 col-lg-3 justify-content-end rounded-4" data-name="Block" style="--grid-item-padding-y: 20px; --grid-item-padding-x: 20px; z-index: 2; grid-area: 1 / 6 / 5 / 9;">
+            <h3>Key <br class="d-none d-lg-inline"/>Milestone</h3>
+            <p>Reaching new heights together</p>
         </div>
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-5 text-center o_cc o_cc4" data-name="Block"
-            style="grid-area: 6 / 7 / 11 / 10; z-index: 4;">
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+        <div class="o_grid_item o_cc o_cc4 g-height-4 g-col-lg-4 col-lg-4 justify-content-end rounded-4" data-name="Block" style="--grid-item-padding-y: 20px; --grid-item-padding-x: 20px; z-index: 4; grid-area: 5 / 6 / 9 / 10; background-image: linear-gradient(135deg, var(--o-color-4) -400%, var(--o-color-1) 100%);">
+            <h3>Innovation <br class="d-none d-lg-inline"/>Hub</h3>
+            <p>Where ideas come to life</p>
         </div>
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-5 text-center o_cc o_cc2" data-name="Block"
-            style="grid-area: 6 / 10 / 11 / 13; z-index: 5;">
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+        <div class="o_grid_item g-col-lg-3 o_cc o_cc2 g-height-4 col-lg-3 justify-content-end rounded-4" data-name="Block" style="--grid-item-padding-y: 20px; --grid-item-padding-x: 20px; z-index: 5; grid-area: 5 / 10 / 9 / 13;">
+            <h3>Community <br class="d-none d-lg-inline"/>Focus</h3>
+            <p>Building connections</p>
         </div>
     </div>
 </template>
 
 <template id="s_masonry_block_reversed_template" groups="base.group_user">
     <div class="row o_grid_mode" data-row-count="10">
         <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-5 text-center o_cc o_cc3" data-name="Block"
```

Comment:
Vertical alignment issue :
![Capture d’écran 2024-07-19 à 17 01 59](https://github.com/user-attachments/assets/3164761e-aaec-4b04-aa4a-2db11bb51cf4)

```suggestion
        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-5 justify-content-center text-center o_cc o_cc3" data-name="Block"
```

Same for the three other blocks :)